### PR TITLE
enable insecure access for kube-proxy

### PR DESCRIPTION
**Pull request by mceloud**

### CODE REVIEW

File: metadata/service/pool/single.yml [hunk 1]
```diff
@@ -12,14 +12,12 @@ parameters:
         host: tcpcloud
       host:
         name: ${linux:system:name}
-      master:
+      apiserver:
         host: ${_param:master_address}
-        apiserver:
-          members:
-            - host: ${_param:master_address}
-        etcd:
```
Comment (pupapaik) [inline metadata/service/pool/single.yml]: Why did you remote etcd hosts?

File: kubernetes/files/calico/network-environment.pool [hunk 2]
```diff
@@ -4,7 +4,7 @@
 DEFAULT_IPV4={{ pool.address }}
 
 # The Kubernetes master IP
-KUBERNETES_MASTER={{ pool.master.host }}
+KUBERNETES_MASTER={{ pool.apiserver.host }}
```
Comment (pupapaik) [inline kubernetes/files/calico/network-environment.pool]: Why not just api instead of apiserver?

Comment (mceloud) [inline kubernetes/files/calico/network-environment.pool]: Reusing naming from kubernetes.master.

File: metadata/service/pool/single.yml (hunk 1 above)
Comment (mceloud) [inline metadata/service/pool/single.yml]: It is not used.

**Comment by pupapaik** [comment]
OK, thanks.

**Pull request merged**